packages/code-du-travail-data/indexing/populate.js

```diff
@@ -110,12 +102,77 @@ async function* cdtnDocumentsGen() {
     })
   );
 
-  logger.info("=== Editorial contents ===");
-  yield getEditorialContents();
+  logger.info("=== Conventions Collectives ===");
+  yield require("@socialgouv/kali-data/data/index.json").map(
+    ({ id, num, title, shortTitle, url, effectif }) => {
+      return {
+        effectif,
+        excludeFromSearch: false,
+        id,
+        idcc: parseIdcc(num),
+        shortTitle,
+        slug: slugify(`${num}-${shortTitle}`.substring(0, 80)),
+        source: SOURCES.CCN,
+        text: `IDCC ${num} ${title}`,
+        title,
+        url,
+      };
+    }
+  );
+
+  logger.info("=== Contributions ===");
+  yield require("@socialgouv/contributions-data/data/contributions.json").map(
+    ({ title, answers, id }) => {
+      const slug = slugify(title);
+      fixReferences(answers.generic);
+      answers.conventions.forEach(fixReferences);
+
+      return {
+        answers: {
+          ...answers,
+          generic: {
+            ...answers.generic,
+            markdown: addGlossary(answers.generic.markdown),
+          },
+        },
+        breadcrumbs: getBreadcrumbs(
+          `/${getRouteBySource(SOURCES.CONTRIBUTIONS)}/${slug}`
+        ),
+        description: (answers.generic && answers.generic.description) || title,
+        excludeFromSearch: false,
+        id,
+        slug,
+        source: SOURCES.CONTRIBUTIONS,
+        text: (answers.generic && answers.generic.text) || title,
+        title,
+      };
+    }
+  );
 
   logger.info("=== Fiches SP ===");
   yield getFichesSP();
 
+  const fichesMT = require("@socialgouv/fiches-travail-data/data/fiches-travail.json");
+
+  logger.info("=== page fiches travail ===");
+  yield fichesMT.map(({ sections, pubId, ...content }) => {
+    return {
+      id: pubId,
+      slug: slugify(content.title),
+      ...content,
+      breadcrumbs: getBreadcrumbs(
+        `/${getRouteBySource(SOURCES.SHEET_MT)}/${content.slug}`
+      ),
+      // eslint-disable-next-line no-unused-vars
+      sections: sections.map(({ description, text, ...section }) => ({
```

Comment:
not sure to get why we have description and text here ?

Reply:
text is the full text of the content for search purposes while description simply is an excerpt to display in the tiles of the search results

Reply:
There is a complicated history with the fiche MT and fiche MT split but I think these are removed here because we don't need them when we query the full page. Section's description is only shown in the splitted fiches and section's text is for searching purposes but we already have the full text. These two are used later in the fiche mt split.

Reply:
ah ok so that's just a way to remove them from the section, thanks